llama-index-integrations/vector_stores/llama-index-vector-stores-redis/pyproject.toml

```diff
@@ -21,18 +21,19 @@ ignore_missing_imports = true
 python_version = "3.8"
 
 [tool.poetry]
-authors = ["Your Name <you@example.com>"]
+authors = ["Tyler Hutcherson tyler.hutcherson@redis.com"]
 description = "llama-index vector_stores redis integration"
 exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-vector-stores-redis"
 readme = "README.md"
-version = "0.1.2"
+version = "0.2.0"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.1"
 redis = "^5.0.1"
+redisvl = "^0.1.2"
```

Comment:
does redisvl already manage its own redis dependency? We could remove the raw redis dep then.

Just want to make sure we don't end up in state where these conflict

Reply:
Good catch, redisvl depends on redis